tests/e2e/program/program/parameter_group/test_sync_parameter_group.py

```
@@ -0,0 +1,63 @@
+pytestmark = [pytest.mark.flaky]
+
+
+@pytest.fixture
+def created_parameter_group(mpt_vendor, program_id, parameter_group_data):
+    service = mpt_vendor.program.programs.parameter_groups(program_id)
+    group = service.create(parameter_group_data)
+    yield group
+    try:
+        service.delete(group.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to delete parameter group {group.id}: {error.title}")  # noqa: WPS421
```

Ruff (0.15.10)

[warning] Invalid rule code in `# noqa`: WPS421

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)